chart/jenkins-operator/templates/service.yaml

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
https://jenkinsci.github.io/kubernetes-operator/docs/getting-started/latest/schema/#github.com/jenkinsci/kubernetes-operator/pkg/apis/jenkins/v1alpha2.JenkinsSpec please look at  `service` and `slaveService` 
The operator creates both services so you have to delete this file and map fields to Jenkins CR.